runner/tests/test_port.py

```diff
@@ -0,0 +1,43 @@
+import os
+from uuid import UUID
+from django.test import TestCase
+from django.conf import settings
+from django.core.management import call_command
+from file_system.models import File
+from runner.models import Port, PortType, Run
+
+class TestPorts(TestCase):
+    # load fixtures for the test case temp db
+    fixtures = [
+    "file_system.filegroup.json",
+    "file_system.filetype.json",
+    "file_system.storage.json",
+    "runner.pipeline.json"
+    ]
+
+    def test_true(self):
+        self.assertTrue(True)
```

Comment:
I am not sure I understand this? Why are we testing tests?

Reply:
It's the first test used when setting up a new `TestCase` to help ensure that your new test case is actually getting run. Especially since the current scripted test runner invocation is cherry picking tests to run instead of running them all automatically, it's easy to forget to update the test runner to include the new tests and see a message of "all tests passed" when in fact the new tests were not run. During development, it starts out as `False` to force test failure, then is switched to `True` once you've ensured the test case is running.

Reply:
My opinion is that this is not needed. You could also delete the test the same way as you switched assert to True. I don't go into how do you do development, but this is just additional test which doesn't give us any information. Considering runner invocation, all tests are fixed on branch feature/fix_locations_in_db_value so you will be able to invoke all the tests really soon. I prefer deleting this.